### PR TITLE
kill: remove the `--table` long option

### PR DESCRIPTION
**Pull request by Managor**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**
2.40.2

### CODE REVIEW

**Comment by Managor** [comment]
Not sure what the manual page is on about, `--table` option does not work.

File: pages/linux/kill.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 - List signal values and their corresponding names (to be used without the `SIG` prefix):
 
-`kill {{-L|--table}}`
+`kill {{-l}}`
```
Comment (spageektti) [inline pages/linux/kill.md:13]:
```suggestion
`kill -l`
```

**Review by spageektti — APPROVED** [review]

**Comment by acuteenvy** [comment]
> Not sure what the manual page is on about, `--table` option does not work.

It works for me.

```
$ kill --table
1 HUP      2 INT      3 QUIT     4 ILL      5 TRAP     6 ABRT     6 IOT      7 BUS      8 FPE      9 KILL    10 USR1    11 SEGV    12 USR2    13 PIPE    14 ALRM    15 TERM    16 STKFLT  17 CHLD
17 CLD     18 CONT    19 STOP    20 TSTP    21 TTIN    22 TTOU    23 URG     24 XCPU    25 XFSZ    26 VTALRM  27 PROF    28 WINCH   29 IO      29 POLL    30 PWR     31 SYS     34 RTMIN   64 RTMAX
$ kill -L
<same output as above>
```
I'm on the same version as you (2.40.2). What does `kill --table` print on your system?

**Comment by Managor** [comment]
```kill --table
-bash: kill: -table: invalid signal specification
```

**Comment by acuteenvy** [comment]
This page documents `kill` from util-linux, but you seem to be using the bash builtin `kill`.
Try `enable -n kill && kill --table`.

I guess we have to edit the page to somehow mention it.

**Comment by Managor** [comment]
That seems to indeed be the case. `enable -n kill && kill --table` works as expected and prints the signals.

**Comment by cyqsimon** [comment]
> This page and `man kill` document `kill` from util-linux, but you seem to be using the bash builtin `kill`. Try `enable -n kill && kill --table`.
> 
> I guess we have to edit the page to somehow mention it.

Very interesting! I did not know that.

Running `kill --help` (the shell builtin) gives the following justification, which does make sense.

    Kill is a shell builtin for two reasons: it allows job IDs to be used
    instead of process IDs, and allows processes to be killed if the limit
    on processes that you can create is reached.

**Comment by Managor** [comment]
I'm thinking that we should pass this PR because the result would be more universal and then create an issue to either think about it later or forget about it.
@acuteenvy Can you create an issue about this?

File: pages/linux/kill.md [hunk 2]
```diff
@@ -10,7 +10,7 @@
 
 - List signal values and their corresponding names (to be used without the `SIG` prefix):
 
-`kill {{-L|--table}}`
+`kill -l`
```
Comment (acuteenvy) [inline pages/linux/kill.md:13]: In the bash builtin, `-L` is equivalent to `-l`.
In util-linux, `-l` prints a list without the signal numbers.
```suggestion
`kill -L`
```

**Review by acuteenvy — APPROVED** [review]

**Comment by rikuyaosawa** [comment]
I had a problem using `kill -L` command, but as it was being discussed here, I was using the shell built in `kill` (in my case zsh), where did not recognize the option `-L`. Using `/bin/kill -L` worked as described.  Gotta deep deeper into zsh to find out more about it.

**Comment by Managor** [comment]
Please post in the open issue. https://github.com/tldr-pages/tldr/pull/13435

This is a closed merge request and your message can easily be lost here.

**Comment by Managor** [comment]
Man, I messed up. The correct issue to comment on this was https://github.com/tldr-pages/tldr/issues/13426

**Pull request merged**